config/samples/perf_v1alpha1_sysbench.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: perf.kubestone.xridge.io/v1alpha1
+kind: Sysbench
+metadata:
+  name: sysbench-sample
+spec:
+  image:
+    name: xridge/sysbench:latest
+    # pullPolicy: IfNotPresent
+    # pullSecret: null
+  options: --threads=2 --time=20
```

Comment:
can we please match fio & iperf's notion of `cmdLineArgs`?

Reply:
It is sysbench specific. I wanted to stick to the original naming.

Reply:
It is sysbench specific. I wanted to stick to the original naming.